packages/zoe/test/unitTests/test-zoe.js

```diff
@@ -134,7 +134,9 @@ test(`E(zoe).getPublicFacet - no instance`, async t => {
   // @ts-expect-error deliberate invalid arguments for testing
   await t.throwsAsync(() => E(zoe).getPublicFacet(), {
     message:
-      /In "getPublicFacet" method of \(ZoeService\): arg 0: .*"\[undefined\]" - Must be a remotable/,
+      // Or pattern of golden error tolerant across versions of endo.
```

Comment:
I know I wrote this horrible sentence, but how about

```suggestion
      // Golden test uses RegExp "Or" pattern to tolerate earlier versions of endo
```

Likewise below